tools/ci/travis-install.sh

```diff
@@ -1,6 +1,11 @@
 #!/bin/bash
 set -ex
 
+# Upgrade pip, pytest, numpy before doing anything else
+pip install --upgrade pip
+pip install --upgrade pytest
+pip install --upgrade numpy
```

Comment:
Why did this block need to be moved up?

Reply:
It seems that some of the pip peripherals aren't available by default on GHA images. In this case, `wheels` isn't added until we upgrade pip. This causes problems in the `vectfit` build.